build-llvm11.md

```diff
@@ -181,13 +171,14 @@ POSIX environment under macOS.  KLEE does not work under x86-32.
    Here's an example:
 
    ```
-/usr/lib/llvm-3.4/include/llvm/Support/CommandLine.h:905: undefined reference to `vtable for llvm::cl::parser<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >'
-CMakeFiles/kleaver.dir/main.cpp.o: In function `main':
-/home/user/programs/klee/klee/tools/kleaver/main.cpp:413: undefined reference to `llvm::error_code::message[abi:cxx11]() const'
-CMakeFiles/kleaver.dir/main.cpp.o: In function `llvm::cl::opt<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, false, llvm::cl::parser<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >::~opt()':
+   /usr/lib/llvm-3.4/include/llvm/Support/CommandLine.h:905: undefined reference to `vtable for llvm::cl::parser<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >'
```

Comment:
I wonder if this is still relevant.  The example shows LLVM 3.4

Reply:
Me as well... shouldn't happen on current systems imho.

Reply:
added a commit that removes this section